scanner/fetcher.go

```diff
@@ -62,8 +64,7 @@ type Fetcher struct {
 	// Current STH of the Log this Fetcher sends queries to.
 	sth *ct.SignedTreeHead
 
-	// TODO(pavelkalinnikov): Consider log.Logger instead.
-	Log func(msg string)
+	Logger *log.Logger
```

Comment:
Maybe unexport it too while we're at it?

Reply:
Done.